test/cql-pytest/conftest.py

```diff
@@ -93,12 +95,30 @@ def this_dc(cql):
 # and automatically deleted at the end. We use scope="session" so that all
 # tests will reuse the same keyspace.
 @pytest.fixture(scope="session")
-def test_keyspace(cql, this_dc):
+def test_keyspace(request, cql, this_dc):
     name = unique_name()
-    cql.execute("CREATE KEYSPACE " + name + " WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }")
+    replication_options = [("class", "NetworkTopologyStrategy"), (this_dc, 1)]
+    if request.config.getoption("replication") == "tablets":
+        replication_options.append(("initial_tablets", 128))
```

Comment:
I don't think 128 is a good default. I'd use 4. This may explain why your tests became very slow - it's a lot of work to create 128 tablets.

Reply:
Good point, we don't want to stress-test scylla with 128 tablets in cql-pytest. It was a good experiment, and it shook out some bugs, but we shouldn't do this all the time. Your test adding-dropping tables is a more concentrated test for this.